monero-outreach-docs/translations/es/quickfacts/Quick-facts_es_printerfriendly.sla

```diff
@@ -785,75 +676,247 @@
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="El tamaño del"/>
+            <ITEXT FONT="MonereoSerif Bold Italic" FONTSIZE="14" CH=" "/>
+            <ITEXT FONT="MonereoSerif Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="bloque "/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MonereoSerif Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="dinámico"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH=" de Monero"/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="cambia basado en el"/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="volumen de transacciones para"/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="proveer menores comisiones y transacciones más rápidas. Un"/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="mayor volumen de transacciones"/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="es igual a un mayor tamaño del"/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="bloque, mientras que uno menor"/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="es igual a un menor tamaño del"/>
+            <para ALIGN="2" LINESP="14"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="bloque."/>
+            <ITEXT FONT="MonereoSerif Bold Italic" FONTSIZE="14" FEATURES="inherit" CH=" "/>
+            <ITEXT FONT="MonereoSerif Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="Bulletproof"/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="14" CH="  o pruebas de"/>
```

Comment:
Double space at the beginning of the line.

Reply:
thanks, fixed :)